PolarisO365.ps1

```diff
@@ -373,44 +369,44 @@ function Get-PolarisO365Users() {
                 };
             )
             "first"     = 100;
-            "id"        = $SubscriptionId;
+            "orgId"        = $SubscriptionId;
             "sortBy"    = "EMAIL_ADDRESS";
             "sortOrder" = "ASC";
         }
     }
     $response = Invoke-RestMethod -Method POST -Uri $endpoint -Body $($payload | ConvertTo-JSON -Depth 100) -Headers $headers
-    $node_array += $response.data.o365Org.childConnection.edges.node
+    $node_array += $response.data.o365Mailboxes.edges

```

Comment:
can do this $node_array += $response.data.o365Mailboxes.edges.node
then don't have to call node when fetching each fields (name, id, userPrincipalName, slaAssignment, effectiveSlaDomainName) below.

Reply:
node is not a sub property of edges, rather each elements in edges collection have node as a field.